Sources/HeliumLogger/HeliumLogger.swift

```diff
@@ -46,9 +46,13 @@ public enum HeliumLoggerFormatValues: String {
     case logType = "(%type)"
     /// The time and date at which the message was logged.
     case date = "(%date)"
+    /// The logging metadata used by SwiftLog.
+    case metadata = "(%metadata)"
+    /// The label of the logger used by SwiftLog.
+    case label = "(%label)"
```

Comment:
Unfortunately, this is a breaking change so would require a 2.0 release of HeliumLogger.  I wonder if there's a way around this?

If we must make a breaking change, I might be tempted to replace this enum with a struct so that any future additions are not breaking.  (To illustrate what I am thinking of, there's a comment I just made on a similar theme in https://github.com/IBM-Swift/Kitura/pull/1468)

Reply:
Yep, that totally makes sense. I put these SwiftLog specific variables in a new enum.
```swift
/// The additional set of substitution "variables" that can be used when formatting the
/// messages to be logged with SwiftLog.
public enum HeliumLoggerSwiftLogFormatValues: String {
    /// The logging metadata used by SwiftLog.
    case metadata = "(%metadata)"
    /// The label of the logger used by SwiftLog.
    case label = "(%label)"
    
    static let All: [HeliumLoggerSwiftLogFormatValues] = [
        .metadata, .label
    ]
}
```

I considered some alternative approaches, but this was the lowest-touch option I could come up with. Happy to discuss other ideas, too. 🙂